casdoorsdk/user.go

```diff
@@ -233,6 +238,57 @@ func GetUserByUserId(userId string) (*User, error) {
 	return user, nil
 }
 
+func UpdateUserPassword(owner, name, oldPassword, newPassword string) (bool, error) {
```

Comment:
Plz keep the same name with API path, `SetPassword`

And, does it allow empty "oldPassword" as the user requested?